components/developer/gdb/gdb.p5m

```diff
@@ -34,9 +38,21 @@ set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file path=usr/bin/gdb
+
+file path=usr/bin/gcore
+# The dependency resolution for python 3.9 does not work as expected. For one the searchpath is different 
+# but even with adapting the searchpath to the correct directories pkg is unable to find the library while ldd
+# and ld.so are able to. This is a bug in pkg so we workaround it for now.
```

Comment:
Could you please share the error message, or log, or whatever showing the problem?  It is very hard to judge without more details.  Thank you.

Reply:
Once you confirm you ran `gmake REQUIRED_PACKAGES` and still got python-37 there (instead of python-39) then I'll look at it.